### DISCUSSION TITLE
No receving Mail on mailhog

### DISCUSSION THREAD

**Discussion by dev13-suthar** [dubinc/dub#1374] [Q&A] [answered]
I have setup the codebase locally earlier after signup new account i was receving the OTP and email on mailhog but now im not getting any emails even though my container is Up

**Comment by devkiran** [comment, marked as answer]
Make sure you have the following set on the `.env`.

```
SMTP_HOST=localhost
SMTP_PORT=1025
SMTP_USER=smtpUser
SMTP_PASSWORD=smtpPassword

RESEND_API_KEY #Remove this
```
